tensorflow/contrib/cmake/README.md

```diff
@@ -165,6 +165,29 @@ Step-by-step Windows build
 
 3. Invoke CMake to create Visual Studio solution and project files.
 
+   **NOTE**: MSVC does not support [the flag `-march=native` defaulted on `CMakeLists.txt`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/cmake/CMakeLists.txt#L78) 
+   so in order to build TensorFlow with MSVC using advanced CPU instructions
+   you will need to edit `CMakeLists.txt` and explicitly define the options
+   as in `/arch:[ARG1|ARG2|ETC]`, e.g.:
```

Comment:
The instruction is confusing I think.
What if we did this to avoid the need for users to edit the cmakefile:

In Cmakefiles.txt, add a new option name can be something like "tensorflow_WIN_CPU_SIMD_OPTIONS", default value to this can be empty.
Then we append this to to the MSVC compilation commands if the build is running on windows.

With the above change, our instructions become this: Just set `-Dtensorflow_WIN_CPU_SIMD_OPTIONS=/arch:AVX`

What do you think?